client/jetpack-cloud/sections/partner-portal/license-preview/index.tsx

```diff
@@ -149,14 +149,12 @@ export default function LicensePreview( {
 
 				<div>
 					{ licenseState === LicenseState.Detached && (
-						<ClipboardButton
-							text={ licenseKey }
-							className="license-preview__copy-license-key"
+						<Button
 							compact
-							onCopy={ onCopyLicense }
+							href={ addQueryArgs( { key: licenseKey }, '/partner-portal/assign-license' ) }
 						>
-							{ translate( 'Copy License' ) }
-						</ClipboardButton>
+							{ translate( 'Assign License' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **4** times:
`translate( 'Unable to assign license.' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-license-portal-license-attachment)